README.md

```diff
@@ -84,15 +84,7 @@ Install 3rd party plugins at your own risk.
 
 Additionally, plugins that come bundled with **`ape`** in the core installation cannot be removed and are part of the **`ape`** core software.
 
-## Projects
-
-When using Ape, you generally will work with a project.
```

Comment:
and maybe keep this sentence too or at least have 1 sentence explaining what we mean by `project`.